src/main/scala/com/kjetland/jackson/jsonSchema/JsonSchemaGenerator.scala

```diff
@@ -157,8 +159,14 @@ case class JsonSchemaConfig
   useMultipleEditorSelectViaProperty:Boolean, // https://github.com/jdorn/json-editor/issues/709
   uniqueItemClasses:Set[Class[_]], // If rendering array and type is instanceOf class in this set, then we add 'uniqueItems": true' to schema - See // https://github.com/jdorn/json-editor for more info
   classTypeReMapping:Map[Class[_], Class[_]], // Can be used to prevent rendering using polymorphism for specific classes.
-  jsonSuppliers:Map[String, Supplier[JsonNode]] // Suppliers in this map can be accessed using @JsonSchemaInject(jsonSupplierViaLookup = "lookupKey")
-)
+  jsonSuppliers:Map[String, Supplier[JsonNode]], // Suppliers in this map can be accessed using @JsonSchemaInject(jsonSupplierViaLookup = "lookupKey")
+  reflection:Option[ScanResult] = None
```

Comment:
This also should be added to the Java API so Java users like me can access it :)